locales/pt-BR/translation.json

```
@@ -0,0 +1,57 @@
+{
+    "Workday Waiver Manager": "Gerenciador de Dia Livre",
+    "Exit": "Sair",
+    "Cut": "Cortar",
+    "Copy": "Copiar",
+    "Paste": "Colar",
+    "Select All": "Selectionar Tudo",
+    "Preferences": "Preferências",
+    "Migrate to flexible calendar": "Migrar para calendário flexível",
+    "Export database": "Exportar base de dados",
+    "Import database": "Importar base de dados",
+    "Clear database": "Limpar base de dados",
+    "Reload": "Recarregar",
+    "Toggle Developer Tools": "Alternar Ferramentas de Desenvolvimento",
+    "TTL GitHub": "TTL GitHub",
+    "Check for updates": "Checar por atualizações",
+    "Send feedback": "Enviar comentários",
+    "About": "Sobre",
+    "Punch time": "Bater ponto",
+    "Show App": "Mostrar App",
+    "Quit": "Fechar",
+    "Export DB to file": "Exportar Base de Dados para Arquivo",
+    "Export": "Exportar",
+    "All Files": "Todos os Arquivos",
+    "Import DB from file": "Importar Base de Dados de Arquivo",
+    "Import": "Importar",
+    "Cancel": "Cancelar",
+    "Yes, please": "Sim, por favor",
+    "No, thanks": "Não, obrigado",
+    "Are you sure you want to clear all the data?": "Tem certeza que você quer limpar todos os dados?",
+    "OK": "OK",
+    "Menu": "Menu",
+    "Edit": "Editar",
+    "View": "Janela",
+    "Help": "Ajuda",
+    "Punched time": "Ponto registrado",
+    "Migrate fixed calendar database to flexible": "Migrar calendário fixo para flexível",
+    "Are you sure you want to migrate the fixed calendar database to the flexible calendar?\n\nThe existing flexible calendar database will be cleared": {
+        "": "Tem certeza que deseja migrar o calendário fixo para o flexível?\n\nO calendário flexível existente vai ser limpo."
```

```suggestion
        "": "Tem certeza que deseja migrar da base de dados do calendário fixo para a do flexível?\n\nA base de dados do calendário flexível existente vai ser limpa."
```
And what is this empty string in the array?

---

I'm not sure. The way this works is that I added the `i18n.t`calls and run TTL. All strings are then send to the translations missing, and for some reason, those are getting there like this. I suspect it has something to either with the length or the `\n`.

Either way, I keep it like that.